geopandas/_dask.py

```diff
@@ -0,0 +1,16 @@
+from dask.dataframe.extensions import make_array_nonempty, make_scalar
+import numpy as np
+import shapely.geometry
+
+from .array import GeometryDtype, from_shapely
+
+
+@make_array_nonempty.register(GeometryDtype)
+def _(dtype):
+    a = np.array([shapely.geometry.Point(i, i) for i in range(2)], dtype=object)
+    return from_shapely(a)
+
+
+@make_scalar.register(GeometryDtype.type)
```

Comment:
Just a question for my education: what does dask do with the registered scalar?

Reply:
`make_scalar` is used for reductions, to get the dtype of the scalar:

```python
In [17]: df['name'].max()
Out[17]: dd.Scalar<series-..., dtype=<U3>
```

Reply:
A GeoSeries doesn't really support reductions, though (at least none of the reductions going through `EA._reduction` that returns a scalar of its own type). But anyway, also no problem in having it I suppose (although it might not be covered)